modules/light-clients/08-wasm/types/errors.go

```diff
@@ -15,4 +15,7 @@ var (
 	ErrWasmSubMessagesNotAllowed = errorsmod.Register(ModuleName, 10, "execution of sub messages is not allowed")
 	ErrWasmEventsNotAllowed      = errorsmod.Register(ModuleName, 11, "returning events from a contract is not allowed")
 	ErrWasmAttributesNotAllowed  = errorsmod.Register(ModuleName, 12, "returning attributes from a contract is not allowed")
+
+	// Migrate client
+	ErrClientNotActive = errorsmod.Register(ModuleName, 14, "client not active")
```

Comment:
Instead of adding a new error here, we can use [the existing one in 02-client](https://github.com/cosmos/ibc-go/blob/52da9be5f802419d8888d49c8cc117fbe4013bfd/modules/core/02-client/types/errors.go#L36).